lib/vsc/utils/rest.py

```diff
@@ -169,8 +170,22 @@ def request(self, method, url, body, headers, content_type=None):
         # censor contents of 'Authorization' part of header, to avoid leaking tokens or passwords in logs
         headers_censored = copy.deepcopy(headers)
         headers_censored['Authorization'] = '<actual authorization header censored>'
```

Comment:
use teh same set intersection magic ?

Reply:
good point, done in https://github.com/hpcugent/vsc-base/pull/328/commits/5b034b91ae32421db0771fd7cfd06850a34f212d